editor/src/core/shared/template-path.ts

```diff
@@ -241,18 +238,28 @@ export function scenePathForPath(path: TemplatePath): ScenePath {
   }
 }
 
-export function elementPathForPath(path: ScenePath): StaticElementPath
-export function elementPathForPath(path: StaticInstancePath): StaticElementPath
-export function elementPathForPath(path: InstancePath): ElementPath
-export function elementPathForPath(path: TemplatePath): ElementPath
-export function elementPathForPath(path: TemplatePath): ElementPath {
-  if (isScenePath(path)) {
```

Comment:
this seems like a hard to test change – I guess nothing fell on you after removing the scenePath from here? :)

Reply:
In the case of scenes it was mostly used for the functionality provided by (the now beautifully named) `instancePathForLastPartOfScenePath`, and is covered in various tests so we should be good here